domains/game/Game.jsx

```diff
@@ -1,26 +1,92 @@
 import React from "react";
-import { Text, View } from "react-native";
+import { Text, View, StyleSheet } from "react-native";
 import PropTypes from "prop-types";
 
+const boardObstacles = [
+    [false, false, false, true, false, false, false],
+    [false, true, false, false, false, true, false],
+    [false, false, true, false, true, false, false],
+    [true, false, false, false, false, false, true],
+    [false, false, true, false, true, false, false],
+    [false, true, false, false, false, true, false],
+    [false, false, false, true, false, false, false],
+];
 const Game = ({ playerName, game }) => {
     return (
-        <View>
+        <View
+            style={{ flex: 1, alignItems: "center", justifyContent: "center" }}
+        >
             <Text>
                 Game {game.id}, you are {playerName}
             </Text>
             {game.players.map((player) => {
                 return (
-                    <Text key={playerName}>
-                        {`${playerName} is ${
+                    <Text key={player.name}>
+                        {`${player.name} is ${
                             player.current ? "playing" : "waiting"
                         }`}
                     </Text>
                 );
             })}
+            <View style={styles.board}>
+                {boardObstacles.map((row) => {
+                    return (
+                        <View
+                            key={"row" + boardObstacles.indexOf(row)}
+                            style={{
+                                flex: 1,
+                                flexDirection: "row",
+                                alignItems: "center",
+                                justifyContent: "center",
+                            }}
+                        >
+                            {row.map((cell) => {
```

Comment:
You can access x here
```suggestion
                            {row.map((cell, x) => {
```